db/seed.js

```diff
@@ -95,5 +95,12 @@ class dataGenerator {
 }
 
 let generator = new dataGenerator();
-let product = generator.generateProduct();
-productDetails.create(product);
+products = [];
+let product;

```

Comment:
I'm curious why we're declaring an empty variable and re-assigning it with every iteration of the loop